examples/deadline/All-In-AWS-Infrastructure-Basic/ts/README.md

```diff
@@ -12,65 +12,79 @@ These instructions assume that your working directory is `examples/deadline/All-
 
 ---
 
-1. Install the dependencies of the sample app:
-```
-yarn install
-```
-2. Change the value in the `deadlineClientLinuxAmiMap` variable in `bin/config.ts` to include the region + AMI ID mapping of your EC2 AMI(s) with Deadline Worker.
-```ts
-// For example, in the us-west-2 region
-public readonly deadlineClientLinuxAmiMap: Record<string, string> = {
-  ['us-west-2']: '<your-ami-id>',
-  // ...
-  };
-```
-3. Create a binary secret in [SecretsManager](https://aws.amazon.com/secrets-manager/) that contains your [Usage-Based Licensing](https://docs.thinkboxsoftware.com/products/deadline/10.1/1_User%20Manual/manual/aws-portal/licensing-setup.html?highlight=usage%20based%20licensing) certificates in a `.zip` file:
-```
-aws secretsmanager create-secret --name <name> --secret-binary fileb://<path-to-zip-file>
-```
-4. The output from the previous step will contain the secret's ARN. Change the value of the `ublCertificatesSecretArn` variable in `bin/config.ts` to the secret's ARN:
-```ts
-public readonly ublCertificatesSecretArn: string = '<your-secret-arn>';
-```
-5. Choose your UBL limits and change the value of the `ublLicenses` variable in `bin/config.ts` accordingly:
-```ts
-public readonly ublLicenses: UsageBasedLicense[] = [ /* <your-ubl-limits> (e.g. UsageBasedLicense.forMaya(10)) */ ];
-```
----
+1.  Install the dependencies of the sample app:
 
-**Note:** The next two steps are optional. You may skip these if you do not need SSH access into your render farm.
+    ```
+    yarn install
+    ```
+2.  Change the value in the `deadlineClientLinuxAmiMap` variable in `bin/config.ts` to include the region + AMI ID mapping of your EC2 AMI(s) with Deadline Worker.
 
----
-6. Create an EC2 key pair to give you SSH access to the render farm:
-```
-aws ec2 create-key-pair --key-name <key-name>
-```
+    ```ts
+    // For example, in the us-west-2 region
+    public readonly deadlineClientLinuxAmiMap: Record<string, string> = {
+      ['us-west-2']: '<your-ami-id>',
+      // ...
+      };
+    ```
+3.  Create a binary secret in [SecretsManager](https://aws.amazon.com/secrets-manager/) that contains your [Usage-Based Licensing](https://docs.thinkboxsoftware.com/products/deadline/10.1/1_User%20Manual/manual/aws-portal/licensing-setup.html?highlight=usage%20based%20licensing) certificates in a `.zip` file:
+
+    ```
+    aws secretsmanager create-secret --name <name> --secret-binary fileb://  <path-to-zip-file>
+    ```
+4.  The output from the previous step will contain the secret's ARN. Change the value of the `ublCertificatesSecretArn` variable in `bin/config.ts` to the secret's ARN:
+
+    ```ts
+    public readonly ublCertificatesSecretArn: string = '<your-secret-arn>';
+    ```
+5.  Choose your UBL limits and change the value of the `ublLicenses` variable in `bin/config.ts` accordingly:
+
+    ```ts
+    public readonly ublLicenses: UsageBasedLicense[] = [ /* <your-ubl-limits> (e.g.   UsageBasedLicense.forMaya(10)) */ ];
```

Comment:
Can this example be also updated for unlimited license, similar to that of python?

Reply:
done